docs/en_US/SupportedFramework_Library.md

```diff
@@ -0,0 +1,51 @@
+# Framework and Library Supported 
+NNI provides Python API, supporting all frameworks and libraries written in python (`version >= 3.5`), and we have implemented many related examples and detailed their principles and running steps.
+
+
+
+## Support Framework:
+
+NNI supports all frameworks, as long as they are written in python. Some common frameworks are listed below, including NNI example tutorials based on them.
```

Comment:
remove this sentence, seems dup with the above one.

Reply:
fixed